Tests/UITests/TestApp/LLMOpenAIRealtime/LLMOpenAIRealtimeTestView.swift

```diff
@@ -0,0 +1,44 @@
+//
+// This source file is part of the Stanford Spezi open source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziLLM
+import SpeziLLMOpenAI
+import SpeziLLMOpenAIRealtime
+import SwiftUI
+
+struct LLMOpenAIRealtimeTestView: View {
+    static let schema = LLMOpenAIRealtimeSchema {
+        LLMOpenAIFunctionWeather()
+    }
+    
+    @LLMSessionProvider(schema: Self.schema) var llm: LLMOpenAIRealtimeSession
+
+    @State var audio = AudioViewModel()
+    
+    var body: some View {
+        VStack {
+            LLMChatView(session: $llm)
```

Comment:
Might make sense to disable the chat input field while an live audio recording is done.